xpra/server/window/window_video_source.py

```diff
@@ -778,7 +784,7 @@ def send_nonvideo(regions=regions, encoding=coding, exclude_region=None, get_bes
         assert not self.full_frames_only
 
         actual_vr = None
-        if vr in regions:
+        if vr in regions or (WINDOW_FOCUS_VIDEO_ENCODER and self.has_focus):
```

Comment:
As above, this looks wrong.

Reply:
also attempting to short-circuit the logic to use nvenc where possible on the active window